core/includes/theme.inc

```diff
@@ -1951,7 +1951,19 @@ function theme_dropbutton_wrapper($variables) {
  */
 function theme_image($variables) {
   $attributes = $variables['attributes'];
-  $attributes['src'] = file_create_url($variables['uri']);
+
+  // Check for missing 'uri', allow D7's 'path' instead.
+  $uri = $variables['uri'];
+  if (!$uri && !empty($variables['path'])) {
+    watchdog('Image',
```

Comment:
I realize that it might make it easier to read, but if I'm not mistaken, coding standards dictate putting function parameters in a single line - we only do the separate line for array keys/values. Also, from a quick look in other instances of `watchdog` the `$type` parameter passed is always lower-case (so `'image'` instead of `'Image'`) and `$variables` parameter should be `NULL` *if message is already translated or not possible to translate.*